src/algebra/category/FinVect/limits.lean

```diff
@@ -58,7 +58,8 @@ end
 def forget₂_creates_limit (F : J ⥤ FinVect k) :
   creates_limit F (forget₂ (FinVect k) (Module.{v} k)) :=
 creates_limit_of_fully_faithful_of_iso
-  ⟨(limit (F ⋙ forget₂ (FinVect k) (Module.{v} k)) : Module.{v} k), by apply_instance⟩
+  ⟨(limit (F ⋙ forget₂ (FinVect k) (Module.{v} k)) : Module.{v} k),
+    FinVect.finite_dimensional_limit F⟩
```

Comment:
Is this necessary?

Reply:
It is if we define `FinVect` as `full_monoidal_subcategory` but with your previous comment I can revert this to `apply_instance`.